pom.xml

```diff
@@ -3,40 +3,41 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>2.11</version>
+    <version>3.43</version>
+    <relativePath />
   </parent>
 
   <artifactId>form-element-path</artifactId>
   <version>1.9-SNAPSHOT</version>
   <packaging>hpi</packaging>
   <name>Form element path plugin</name>
-  <description>Introduce unique path attributes for UI controls to facilitate Jenkins UI testing.</description>
-  <url>https://wiki.jenkins-ci.org/display/JENKINS/Form+Element+Path+Plugin</url>
+  <description>Adds Path designators to form elements to simplify Selenium testing of Jenkins.</description>
```

Comment:
FWIW unused in Jenkins project infra, as https://github.com/jenkinsci/form-element-path-plugin/blob/master/src/main/resources/index.jelly is given precedence.

Reply:
Yup, just for consistency. The purpose of the commit was just to trigger a new build anyway.